docs/references/content-as-code.mdx

```diff
@@ -0,0 +1,47 @@
+# Content as code
+
+You can download your charts and dashboards as code, make changes, then upload your changes back to your Lightdash project using the CLI. Content as code is also useful for creating new Lightdash projects where you want the content to be copied from an existing project and added to your new project.
+
+## `lightdash download` downloads content as code
+
+From the [Lightdash CLI](/references/lightdash-cli), you can use the command `lightdash download` to download all of the charts and dashboards from your Lightdash project as code. All of the charts and dashboards will be written as .yml files to a `lightdash` directory inside of your dbt project. 
+
+#### Running `lightdash download` will overwrite any changes you have locally
```

Comment:
should we create a section for `use cases` ? or make this a toggle

Reply:
we can explain first how download/upload works, and then group these usecases together